src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -60,15 +240,58 @@ function Analysis(props: ComponentProps) {
                     </QuickActionButton>
                     <QuickActionButton
                         className={styles.button}
+                        onClick={handleDeleteAnalysis}
                     >
                         <Icon name="delete" />
                     </QuickActionButton>
                 </div>
             )}
+            contentClassName={styles.pillarContent}
         >
-            Content
+            <div className={styles.content}>
+                Content
+            </div>
+            <div
+                className={styles.pillarAnalyses}
+            >
+                <Button
+                    className={styles.accordionButton}
+                    icons={(
+                        <Icon name={isExpanded
+                            ? 'chevronUp'
+                            : 'chevronDown'}
+                        />
+                    )}
+                    onClick={handleClick}
+                >
+                    {_ts('analysis', 'pillarAnalysisCount', { count: pillarCount })}
+                </Button>
+                {isExpanded && (
+                    <>
+                        <div className={styles.pillarAnalysisContent}>
+                            <ListView
+                                className={styles.pillarList}
+                                data={analysisPillar}
+                                keySelector={keySelector}
+                                renderer={AnalysisPillarRenderer}
+                                rendererParams={analysisPillarRendererParams}
+                                pending={pillarPending}
+                            />
+                        </div>
+                        {isDefined(pillarCount) && pillarCount >= MAX_ITEMS_PER_PAGE && (
```

Comment:
here setting pillarCount in state is not useful. Instead we could get the count directly as 
`const [
        pillarPending,
        pillarResponse,
        ,
        pillarGetTrigger,
    ] = useRequest<MultiResponse<AnalysisPillars>>(`
and use it as 
```suggestion
                        {pillarResponse && pillarResponse.count > MAX_ITEMS_PER_PAGE && (
```
here > should be used instead of >=. Do you know why?

Reply:
Yeah, because MAX_ITEMS_PER_PAGE will be in a single page and no pagination is needed. That was my bad! Thanks.